### PR TITLE
Set up extension bundling

### PR DESCRIPTION
**Pull request by Ge11ert**
This PR adds `esbuild` to bundle extension before publishing.

It heavily depends on Crafter update in which parsers resolution should be refactored. See: https://github.com/funbox/crafter/pull/10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ we created this implementation of Language Server.
 
 ### Build extension from source
 
-* Run `npx vsce package` in the root directory.
+* Run `npx @vscode/vsce package` in the root directory.
```
Comment (Ge11ert) [inline README.md:38]: `vsce` package is now deprecated and should be replaced with `@vscode/vsce`. See https://www.npmjs.com/package/vsce

File: server/package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "node": "*"
   },
   "dependencies": {
-    "@funboxteam/crafter": "4.1.0",
+    "@funboxteam/crafter": "file:./funboxteam-crafter-4.2.0.tgz",
```
Comment (Ge11ert) [inline server/package.json:9]: At now, I use local Crafter package with updates from https://github.com/funbox/crafter/pull/10

Thus, this PR cannot be merged right now. But it still can be reviewed.

Comment (igoradamenko) [reply server/package.json:9]: Published @funboxteam/crafter@4.1.1.

Comment (Ge11ert) [reply server/package.json]: I've replaced crafter with public version. It seems, bundled extension (locally packed) works as expected

File: README.ru.md [hunk 3]
```diff
@@ -135,6 +135,17 @@ File → Preferences → Settings → Extensions → API Blueprint → Entry Poi
 этом результаты вывода необходимо искать в окне `Extension Development Host` в
 панели `Output` - `API Blueprint Language Server`.
 
+### Сборка публичной версии
+
+Перед публикацией исходники расширения собираются в бандлы (отдельный бандл для `client`, отдельный бандл для `server`)
+с помощью esbuild.
+
+Чтобы собрать минифицированную версию бандла (аналогично той, которая будет опубликована),нужно выполнить `npm run esbuild-min`.
```
Comment (igoradamenko) [inline README.ru.md:143]: > опубликована),нужно

Space.

Comment (igoradamenko) [reply README.ru.md]: Never mind, I've fixed.

**Pull request merged**